### ISSUE TITLE
Support for High Sierra?

### ISSUE THREAD

**Issue by jby** [chilcote/vfuse#43]
Hey Joe,

You planning on supporting High Sierra (10.13)?

I just tried it and got this:

```bash
Mounting osx-10.13-17A365.apfs.dmg
macOS version is 10.13
This macOS version is not supported: 10.13
Unmounting Base OS DMG
```

**Comment by chilcote** [comment]
I updated it to work with Fusion 10 and hw version 14. If you want to use the latest commit with Fusion 8.5.x then try passing the `--hw-version 12` argument. Otherwise use the latest release posted as that's not been updated yet.

**Comment by jby** [comment]
Well, my /usr/local/vfuse/vfuse only contains 747 rows and has that statement on row 667.

```bash
pkgutil --pkg-info com.chilcote.vfuse
package-id: com.chilcote.vfuse
version: 1.0.3
volume: /
location: /
install-time: 1496259632
```

I'll try cloning your repo and try that instead

**Comment by clburlison** [comment]
Your package says version 1.0.3. The latest package version is 1.0.5. Downloading from the releases tab will likely be easier then cloning the repo.

**Comment by jby** [comment]
Yeah, well - the problem with having two computers to maintain...

Sorry @chilcote for wasting your time. I have an updated version on my work-WS, and **now** on my laptop as well... 😊 

I just remembered that I installed vfuse last week, which I did - on my WS at the office, so when I ran into problems on my laptop working from home, I was sure it was recently installed. But now I know better. 1.0.5 works just fine, thanks @clburlison !
